add-on/chrome/bootstrap.js

```diff
@@ -343,6 +345,42 @@ var WindowListener = {
 
         this.maybeAddCopyPanel();
         this.updateToolbarState();
+
+        this.addSidebar();
+      },
+
+      /* XXX make this into it's own object */
```

Comment:
These need to change to `XXX akita`... probably worth a general check through the patch.